docs/core/diagnostics/lldb-linux.md

```diff
@@ -49,7 +49,7 @@ To install the LLDB packages:
 
 ```console
     apk update
-    apk add lldb
+    apk add lldb py3-lldb
```

Comment:
Py-lldb should only be needed for testing the diagnostic repo. The extension itself is native.

Reply:
It's for end-user:

```sh
$ docker run -it alpine
$ apk add lldb
$ lldb # note: just empty lldb command execution will give the warning
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'lldb'
```

After this error-as-warn, it works; so basically it's sort of a prereq of Alpine itself for clean execution.

Reply:
Odd - but I can see it. Seems like a packaging issue, but then this LGTM